hlint/hlint.dhall

```diff
@@ -882,4 +882,18 @@ in [ rule.Arguments { arguments =
    , warnLifted "putBSLn" "x"
    , warnLifted "putLBS" "x"
    , warnLifted "putLBSLn" "x"
+
+   ------------
+   -- Extra
+   ------------
+
+   -- Tuple
+   , hintNote "fmap (,a) (f a)"         "traverseToFst f a" "Use `traverseToFst`"
+   , hintNote "fmap (flip (,) a) (f a)" "traverseToFst f a" "Use `traverseToFst`"
+   , hintNote "(,a) <$> f a"            "traverseToFst f a" "Use `traverseToFst`"
+   , hintNote "flip (,) a <$> f a"      "traverseToFst f a" "Use `traverseToFst`"
+   , hintNote "fmap (a,) (f a)"    "traverseToSnd f a" "Use `traverseToSnd`"
+   , hintNote "fmap ((,) a) (f a)" "traverseToSnd f a" "Use `traverseToSnd`"
+   , hintNote "(a,) <$> f a"       "traverseToSnd f a" "Use `traverseToSnd`"
+   , hintNote "(,) a <$> f a"      "traverseToSnd f a" "Use `traverseToSnd`"
```

Comment:
I can't think of any non-contrived rules for `traverseBoth`, especially ones that wouldn't involve `curry` or something equivalent to it.

Reply:
Existing examples are already good enough :+1: Don't need to support all crazy combinations.